src/app/page.js

```diff
@@ -1,9 +1,135 @@
-import Image from 'next/image'
-
+"use client"
+import Image from 'next/image';
+import { useState } from "react";
+import { useRouter } from 'next/navigation';
 export default function Home() {
+  const [searchTerm, setSearchTerm] = useState('');
+  const router = useRouter();
+
+  const handleSearch = (e) => {
+    e.preventDefault();
+    router.push(`/search?term=${searchTerm}`);
+    setSearchTerm('');
+  };
   return (
-   <div className=''>
```

Comment:
so basically you need to have this kind of empty div, then components can be wrapped inside.